incubator/patroni/templates/statefulset-patroni.yaml

```diff
@@ -77,7 +83,7 @@ spec:
   - metadata:
       name: pgdata
       annotations:
-        volume.alpha.kubernetes.io/storage-class: default
+        volume.alpha.kubernetes.io/storage-class: {{default "default" .Values.StorageClass }}
```

Comment:
This should to be `beta` then. Also, there's a pattern in other charts for this matter that we may want to use here as well.